config/code_schools.yml

```diff
@@ -448,3 +448,13 @@
   va_accepted: true
   full_time: true
   hardware_included: false
+  
+  - name: Devry Bootcamp
```

Comment:
Fixed and resubmitted. Should be fixed now. I no longer see the indentation issue on my end. How about you?